> Tests / Windows (free-threading) / Build and test (arm64, switch-case) (pull_request): Failing after 25m

test_rollover_at_midnight() failed: see https://github.com/python/cpython/issues/84649#issuecomment-4831009636

```
FAIL: test_rollover_at_midnight (test.test_logging.TimedRotatingFileHandlerTest.test_rollover_at_midnight)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\a\cpython\cpython\Lib\test\test_logging.py", line 6749, in test_rollover_at_midnight
    self.assertIn(f'testing2 {i}', line)
    ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: 'testing2 0' not found in '2026-06-30 14:20:24,102 testing1 0\n'
```

--- a/Lib/test/pythoninfo.py
+++ b/Lib/test/pythoninfo.py
@@ -310,6 +310,7 @@ def collect_os(info_add):
         "COLUMNS",
         "COMPUTERNAME",
         "COMSPEC",
+        "CONTAINER",
         "CPP",
         "CPPFLAGS",
         "DISPLAY",
@@ -369,9 +370,6 @@ def collect_os(info_add):
         "_PYTHON_SYSCONFIGDATA_NAME",
         "_PYTHON_SYSCONFIGDATA_PATH",
         "__PYVENV_LAUNCHER__",
-
-        # Lower case variables
-        "container",
 
         # Sanitizer options
         "ASAN_OPTIONS",

